app/page.tsx

```diff
@@ -1,27 +1,30 @@
-import App from '@/app/app';
-import LDButton from '@/app/LDButton';
-import { getLDContext } from '@/app/utils';
-import { useLDClientRsc } from '@/ld/server';
-import Link from 'next/link';
+import HelloClient from './components/helloLDClient';
+import HelloRSC from './components/helloLDRSC';
 
-// Server Components must useLDClientRsc for evaluation.
+// GOTCHA: page components must be async. Otherwise the ssr cache may not be initialized correctly.
```

Comment:
This comment is no longer correct. Page components can be async or not async. The only requirements are:

* Server Components must use `useLDClientRsc`
* Client Components must use `useLDClient`

```suggestion
```